docs/src/index.md

```diff
@@ -268,6 +268,228 @@ df2 = @eachrow df begin
 end
 ```
 
+## Row-wise transformations with `@byrow`
```

Comment:
I think the second paragraph still applies: it would be nice to start with a sentence or two saying that `@byrow` allows writing code that is applied to each row instead of having to vectorize it.